configs/rtmdet/rtmdet_l_8xb32-300e_coco.py

```diff
@@ -47,18 +47,18 @@
         share_conv=True,
         pred_kernel_size=1,
         norm_cfg=dict(type='SyncBN'),
-        act_cfg=dict(type='SiLU')),
+        act_cfg=dict(type='SiLU', inplace=True)),
     train_cfg=dict(
         assigner=dict(type='DynamicSoftLabelAssigner', topk=13),
         allowed_border=-1,
         pos_weight=-1,
         debug=False),
     test_cfg=dict(
-        nms_pre=1000,
+        nms_pre=30000,
```

Comment:
We can add comments in the config to indicate why and its outcome so that users can better understand it.